docs/model/object/ownership/index.md

```diff
@@ -18,23 +18,27 @@ To make the current owner easier to find from the object's description, the owne
 
 __Example:__
 
-The current owner of the painting is a Museum.
+The current owner of Spring is the Getty.
 
 ```crom
-top = vocab.Painting(ident="auto int-per-segment",label="Painting", art=1)
-top.current_owner = vocab.MuseumOrg(label="Museum")
+top = model.HumanMadeObject(ident="spring/6", label="Jeanne (Spring) by Manet")
+top.current_owner = model.Group(ident="https://vocab.getty.edu/ulan/500115988", label="Getty Museum")
 ```
 
 ## Custody
 
 Similarly, the current custodian of the object should be referenced with the `transferred_custody_to` in the most recent `TransferOfCustody`, in the same way as the current owner is referenced from the most recent `Acquisition`. There is an equivalent property to `current_owner` for custody, which is `current_custodian`.  This is described in more detail in the section on [custody](/model/provenance/custody).
 
-Note that objects are owned by legal entities, such as people or organizations. If there is a department or person that is not the owner but is responsible for the object, then that actor is the `current_custodian`.
+If there is a department or person that is not the owner but is responsible for the object, then that actor is the `current_custodian`. Similarly, loans between organizations (either temporary or permanent) transfer custody rather than ownership.
+
+__Example:__
+
+The Night Watch is owned by the City of Amsterdam, and on permanent loan to the Rijksmuseum.
 
 ```crom
-top = vocab.Painting(ident="auto int-per-segment",label="Painting", art=1)
-top.current_owner = vocab.MuseumOrg(label="Museum")
-top.current_custodian = vocab.Department(label="Paintings Department")
+top = vocab.Painting(ident="nightwatch/14",label="Painting", art=1)
+top.current_owner = model.Group(ident="amsterdam_govt", label="City of Amsterdam Governing Body")
```

Comment:
In the (temporary) absence of a Rijksmuseum URI for this entity, could we identify the owner with this label & URI? [the City of Amsterdam](https://www.wikidata.org/entity/Q9899). Would be happy to amend the PR.

Reply:
I did find that wikidata entry, and was another use case I had in mind when I wrote https://github.com/linked-art/linked.art/issues/461#issuecomment-1877808973. E.g. is that wikidata entry the Place of Amsterdam, or the Group of Amsterdam? It seems like it's both -- it's located in North Holland (so part_of to a Place) has coordinates, shares borders with other places, and contains settlements of Zunderdorp (etc) (so parts) ... but has a head  (Femke Halsema as a role within a Group), twinned administrative bodies (groups?) and importantly! is claimed as the owner of objects like The Jewish Bride and ... The Night Watch.

So ... for the ownership of The Night Watch, we mean Q9899 ... but only as a Group, not as a Place.

Meaning we would assert:

```
top.current_owner = model.Group(ident="https://www.wikidata.org/entity/Q9899", label="City of Amsterdam")
```

We might then also say that Rembrandt had a residence in the City of Amsterdam (somewhat shorthand form):

```
top = model.Person(ident="ulan:rembrandt")
top.residence = model.Place(ident="https://www.wikidata.org/entity/Q9899", label="City of Amsterdam")
```

And now we have an even bigger pain point to deal with.

Thoughts?

Reply:
Yup, Q9899 is a bit groupy-and-placey. But there is also [Q727](http://www.wikidata.org/entity/Q727) which is only `Place`.

In our own Linked Art implementation we would have our own URIs for these separate `Place` and `Group` entities. Although we're not ready yet, soon we would be able to state something like:

`Place`

```json
{
  "id": "https://id.rijksmuseum.nl/23041",
  "type": "Place",
  "_label": "Amsterdam",
  "equivalent": [
    {
      "id": "http://www.wikidata.org/entity/Q727",
      "type": "Place",
      "_label": "Amsterdam"
    }
  ]
}
```

`Group`

```json
{
  "id": "https://id.rijksmuseum.nl/210967",
  "type": "Group",
  "_label": "Gemeente Amsterdam",
  "classified_as": [
    {
      "id": "https://vocab.getty.edu/aat/300252609",
      "type": "Type",
      "_label": "governing bodies"
    }
  ],
  "equivalent": [
    {
      "id": "http://www.wikidata.org/entity/Q9899",
      "type": "Group",
      "_label": "City of Amsterdam"
    }
  ],
  "carried_out": [
    {
      "type": "Activity",
      "took_place_at": [
        {
          "id": "https://id.rijksmuseum.nl/23041",
          "type": "Place",
          "_label": "Amsterdam"
        }
      ]
    }
  ]
}
```

.. after which we can say that Rembrandt is resident in the `Place`, which is related to the `Group` which donated the `HumanMadeObject`

Reply:
Good that there's Q727!  (c.f. painting (type) vs painting (process)) And yes, that's what we do in LUX as well -- everything gets its own URI, with equivalents. Some URIs in LUX have the same equivalent with different classes, for example:

https://lux.collections.yale.edu/view/group/de48ab54-2bb4-4334-be20-4c2912aae220 is the University of Cambridge as a Group.

https://lux.collections.yale.edu/view/place/014bc020-5b74-4cde-8a72-e7465a3bc09d is the University of Cambridge as a Place.

Both have https://www.wikidata.org/wiki/Q35794 as an equivalent, meaning that we assert both 

```
https://www.wikidata.org/wiki/Q35794 rdf:type crm:Group
https://www.wikidata.org/wiki/Q35794 rdf:type crm:Place
```

So for /this/ issue ... yes, lets update to Q9899.